scripts/utils/APIUtils.js

```diff
@@ -28,40 +28,42 @@ function extractPagination(response) {
   return {
     nextPageUrl: nextLink.split(';')[0].slice(1, -1)
   };
-}
+};
 
-export function request(endpoint) {
-  if (endpoint.indexOf(API_ROOT) === -1) {
-    endpoint = API_ROOT + endpoint;
-  }
+export default {
```

Comment:
Why change these folks to be `export default`? They are never needed as a single object.